Sources/SwiftFormatPrettyPrint/Comment.swift

```diff
@@ -61,13 +62,13 @@ struct Comment {
     }
   }
 
-  func print(indent: Int) -> String {
+  func print(indent: [Indent]) -> String {
```

Comment:
I intentionally tried to keep modifications to `Comment` to a minimum; this function could be improved more under the new model, but it would involve processing the comments in a slightly different way so I wanted to keep that as a cleanup task for later instead of increasing the surface of this PR.